pkg/dao/templates.go

```diff
@@ -115,16 +119,16 @@ func (t templateDaoImpl) insertTemplateRepoConfigs(tx *gorm.DB, templateUUID str
 
 	err := tx.Clauses(clause.OnConflict{
 		Columns:   []clause.Column{{Name: "template_uuid"}, {Name: "repository_configuration_uuid"}},
-		DoNothing: true,
+		DoUpdates: clause.AssignmentColumns([]string{"deleted_at"}),
```

Comment:
would this cause the deleted_at column to be nulled out in the case where a user removes a repo and then adds it back?